core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.spec.ts

```diff
@@ -615,7 +615,8 @@ describe('Translation Modal Component', () => {
 
     describe('when alt text is not changed in copied images', () => {
       it('should not submit the translation', () => {
-        component.textToTranslate = '<oppia-noninteractive-image alt-with-' +
```

Comment:
Don't you still want to set `component.textToTranslate` to make sure the code is not using that for validation anymore? Ditto below. We should also have a test replicating the bug.

Reply:
Done